library/alloc/src/vec.rs

```diff
@@ -2566,7 +2566,7 @@ __impl_slice_eq1! { [const N: usize] Vec<A>, &[B; N], #[stable(feature = "rust1"
 //__impl_slice_eq1! { [const N: usize] Cow<'a, [A]>, &[B; N], }
 //__impl_slice_eq1! { [const N: usize] Cow<'a, [A]>, &mut [B; N], }
 
-/// Implements comparison of vectors, lexicographically.
+/// Implements comparison of vectors, [lexicographically](https://doc.rust-lang.org/std/cmp/trait.Ord.html#lexicographical-comparison).
```

Comment:
```suggestion
/// Implements comparison of vectors, [lexicographically](../../std/cmp/trait.Ord.html#lexicographical-comparison).
```

Reply:
Tested all the links that I changed, and it worked.